src/framework/mpas_constants.F

```diff
@@ -1,4 +1,4 @@
-! Copyright (c) 2013,  Los Alamos National Security, LLC (LANS)
+! Copyright (c) 2013-2020  Los Alamos National Security, LLC (LANS)
```

Comment:
I think we should update the copyright date in another PR. One reason would be to have consistent dates, for instance the [copyright date in the top level LICENSE](https://github.com/MPAS-Dev/MPAS-Model/blob/d5a656a90e0fb9a40d40547d4c4f5b832e139518/LICENSE#L1) is from `2013-2019`.

 Perhaps we could have a PR that updates the copyright date across the code base.

Reply:
That sounds reasonable. For some reason I was inspired to update the copyright date when I was making the changes to this file, but I generally agree that PRs should not make extraneous code changes.